spinn_machine/json_machine.py

```diff
@@ -157,9 +157,11 @@ def _find_virtual_links(machine):
             destination = machine.get_chip_at(
                 link.destination_x, link.destination_y)
             inverse_link = destination.router.get_link(inverse_id)
-            assert inverse_link.destination_x == chip.x
-            assert inverse_link.destination_y == chip.y
-            virtual_links_dict[destination].append(inverse_link)
+            # Because some virtual chips are branched at an FPGA, only do this
```

Comment:
I suppose this comment needs revisiting as virtual chips are no longer used?

Reply:
Fixed by removing virtual chips all round